lib/gcpspanner/chromium_daily_usage_stat.go

```diff
@@ -16,29 +16,117 @@ package gcpspanner
 
 import (
 	"context"
+	"errors"
+	"math/big"
 	"time"
+
+	"cloud.google.com/go/civil"
+	"cloud.google.com/go/spanner"
+	"google.golang.org/api/iterator"
 )
 
-// ChromiumDailyUsageStatsWithTime contains usage stats for a feature at a given time.
-type ChromiumDailyUsageStatWithTime struct {
-	Date  time.Time `spanner:"Date"`
-	Usage *int      `spanner:"Usage"`
+const (
+	getChromiumDailyUsageBaseRawTemplate = `
+	SELECT
+		dchm.Day as Date,
+		dchm.Rate as Usage
+	FROM DailyChromiumHistogramMetrics dchm
+	LEFT OUTER JOIN WebFeatureChromiumHistogramEnumValues wfchev
+	ON wfchev.ChromiumHistogramEnumValueID = dchm.ChromiumHistogramEnumValueID
+	JOIN WebFeatures wf
+	ON wfchev.WebFeatureID = wf.ID
+	WHERE wf.FeatureKey = @featureKey
+	AND TIMESTAMP(dchm.Day) >= @startAt AND TIMESTAMP(dchm.Day) < @endAt
+{{ if .PageFilter }}
+ 	{{ .PageFilter }}
+{{ end }}
+ 	ORDER BY Date DESC, Usage DESC LIMIT @pageSize`
```

Comment:
For a given feature, there aren't multiple usage rates per day that we need to break the tie.
```suggestion
 	ORDER BY Date DESC LIMIT @pageSize`
```

Reply:
LGTM with one nit.